wsdl/ver10/schema/onvif.xsd

```diff
@@ -1865,6 +1865,36 @@ decoding .A decoder shall decode every data it receives (according to its capabi
 		<xs:complexContent>
 			<xs:extension base="tt:ConfigurationEntity">
 				<xs:sequence>
+					<xs:element name="Encoding" type="xs:string" minOccurs="0">
+						<xs:annotation>
+							<xs:documentation>Audio Media Subtype for the audio format. When configured, device can receive audio only in this format. For definitions see tt:AudioEncodingMimeNames and <a href="https://www.iana.org/assignments/media-types/media-types.xhtml#audio"> IANA Media Types</a>.</xs:documentation>
```

Comment:
"When configured, device can receive audio only in this format." - Better to be a complement if device can for example  RTP PT 0 mulaw, which may not be necessary to specify by decoder options? Should this be a list?